app/src/main/java/chat/rocket/android/members/adapter/MembersAdapter.kt

```diff
@@ -1,25 +1,24 @@
 package chat.rocket.android.members.adapter
 
-import android.view.View
+import android.util.Log
+import android.view.MenuItem
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import chat.rocket.android.R
+import chat.rocket.android.members.presentation.MembersPresenter
 import chat.rocket.android.members.uimodel.MemberUiModel
-import chat.rocket.android.util.extensions.content
 import chat.rocket.android.util.extensions.inflate
-import kotlinx.android.synthetic.main.avatar.view.*
-import kotlinx.android.synthetic.main.item_member.view.*
 
-class MembersAdapter(
-    private val listener: (MemberUiModel) -> Unit
-) : RecyclerView.Adapter<MembersAdapter.ViewHolder>() {
+class MembersAdapter(private val listener: (MemberUiModel) -> Unit, presenter: MembersPresenter?, private val isOwner: Boolean, private val isMod: Boolean) :
+    RecyclerView.Adapter<ViewHolder>() {
     private var dataSet: List<MemberUiModel> = ArrayList()
+    private val enableActions: Boolean = true
 
-    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MembersAdapter.ViewHolder =
-        ViewHolder(parent.inflate(R.layout.item_member))
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder =
+        ViewHolder(parent.inflate(R.layout.item_member), actionsListener, isOwner, isMod)
```

Comment:
`isModerator` sounds better than `isMod`. You have even created a function with that name on `ChatRoomPresenter.kt` for instance.